doc/user/content/overview/architecture.md

```diff
@@ -57,20 +57,20 @@ Broadly, there are three classes of statements in Materialize:
 
 When Materialize receives a `CREATE SOURCE...` statement, it connects to some
 destination to read data. In the case of streaming sources, it attempts to
-connect to a Kafka stream, which it plumbs into its local instance of
-Differential. You can find more information about how that works in the
+connect to a Kafka stream, which it plugs into its local instance of
```

Comment:
do we want to leave room open here for connecting to non-Kafka streams?

Reply:
Yes, that seems appropriate. There are several non-Kafka sources. I'm also a bit confused about "plugging into its local instance of Differential". This isn't new text .. but it's weird.